docs/learn/court.md

```diff
@@ -6,8 +6,298 @@ title: Decentralized Court
+participate in the voting system. The court uses a plurality voting system,
+meaning the outcome with the most votes wins. Each market is associated with one
+inner court, which can be appealed multiple times if someone believes the
+jurors' decision is unjustified.
+
+### Global Disputes as the last Instance
+
+If the number of appeals reaches a certain threshold (currently three appeals)
+or if during the appeal period (or `dispute_duration`) the total `unconsumed`
+stake (see “The selection algorithm”) is below the necessary draw weights (see
+“Calculating necessary draw weights”), the global dispute system can be
+initiated by anyone. This global dispute system allows all ZTG token holders to
+lock funds based on their beliefs. The outcome with the most locked balance wins
+and serves as the final outcome for traders to rely on for redemption. Global
+dispute voting participants have all of their funds unlocked after the winner is
+determined.
+
+### Joining the Court
+
+You can either join the court (extrinsic `join_court`) as an active juror, who
+is responsible for voting, or be a delegator and delegate (extrinsic `delegate`)
+funds to active jurors. It is important to note that the court pool, which
+contains all jurors and delegators, is bounded in size (config parameter
+`MaxCourtParticipants`). If the court pool is full, the lowest-staked
+participant is about to be replaced by a new and higher-staked participant
+(juror or delegator) account id. For this reason, the court pool is sorted by
+the staked amount of each participant. This comes in handy in order to use
+binary searches for items inside the court pool. Court participants can increase
+their stake at any time by calling `join_court`or `delegate` with a higher
+amount than the previous call to these functions. To decrease the stake, it’s
+necessary to exit the court (see ”Exiting court”). To update the pool item
+associated to a court participant, there are two binary searches, the first for
+finding the pool item (the search key is a tuple of stake and the participant
+account id) and with the previous data and the second for inserting the updated
+data inside the pool again.
+
+For the `delegate` extrinsic, the function argument `delegations` contains a
+list of account ids. The dispatch function ensures that the list of
+`delegations` contains actually account ids which are actively participating
+jurors. The list needs to contain at least one account and all account ids need
+to be unique.
+
+### Exiting the Court
+
+Each juror and delegator can exit the court system to retrieve their remaining
+funds. If the juror or delegator is still actively involved in inner court
+cases, only the unused (non-active) funds are returned. The unused stake is
+equivalent to the stake that was not already selected by the selection algorithm
+(see “The selection algorithm”). In order to exit the court and get the funds
```

Comment:
The title here needs to be capitalized properly. You could also add a link to the section.

Reply:
https://github.com/zeitgeistpm/documentation/pull/81/commits/a19875040e67869b6cc091dad09edd5163eef651